### DISCUSSION TITLE
Can I have only some of selected blocks in sidebar editor

### DISCUSSION THREAD

**Discussion by sagar-vaghela** [GrapesJS/grapesjs#5676] [Q&A]
In side editor, we have blocks for '1 column', '2 column', 'text', etc.

We can use this in our code by creating a div with the id that is given config file.

But, for now I only want the block for 'text'.
How do I do it?

**Comment by artf** [comment]
Start by reading the [documentation](https://grapesjs.com/docs/modules/Blocks.html)
